tests/cases/conformance/decorators/class/decoratedClassFromExternalModule.ts

```diff
@@ -0,0 +1,9 @@
+// @target: es6
```

Comment:
Added in https://github.com/Microsoft/TypeScript/commit/d87e80b65b75eecbcd0e08a85b8a25abb28cae7a